sw/device/lib/testing/test_framework/ottf_main.c

```diff
@@ -94,13 +94,12 @@ void log_status(const char *prefix, status_t status) {
   int32_t arg;
   const char *message;
   char mod_id[4];
-  if (status_ok(status)) {
-    LOG_INFO("%skOk(%u)", prefix, status.value);
-  } else if (status_extract(status, &message, &arg, mod_id)) {
+  // FIXME this is an incorrect usage of status_extract
```

Comment:
?

Reply:
I forgot to remove that comment, that was a todo to myself.